chat-android/src/main/java/com/ably/chat/Presence.kt

```diff
@@ -40,21 +47,21 @@ interface Presence : EmitsDiscontinuities {
      * @param {PresenceData} data - The users data, a JSON serializable object that will be sent to all subscribers.
      * @returns {Promise<void>} or upon failure, the promise will be rejected with an {@link ErrorInfo} object which explains the error.
      */
-    suspend fun enter(data: PresenceData?)
+    suspend fun enter(data: PresenceData? = null)
 
     /**
      * Method to update room presence, will emit an update event to all subscribers. If the user is not present, it will be treated as a join event.
      * @param {PresenceData} data - The users data, a JSON serializable object that will be sent to all subscribers.
      * @returns {Promise<void>} or upon failure, the promise will be rejected with an {@link ErrorInfo} object which explains the error.
```

Comment:
Same goes for all other documentation where promise is mentioned as per https://github.com/ably-labs/ably-chat-kotlin/pull/42/files#r1840393426